cmd/ocm-backplane/config/get_test.go

```diff
@@ -0,0 +1,50 @@
+package config
+
+import (
+	"fmt"
+
+	"github.com/golang/mock/gomock"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	cmv1 "github.com/openshift-online/ocm-sdk-go/clustersmgmt/v1"
+	"github.com/openshift/backplane-cli/pkg/ocm"
+	ocmMock "github.com/openshift/backplane-cli/pkg/ocm/mocks"
+	"github.com/spf13/cobra"
+)
+
+var _ = Describe("get command", func() {
+
+	var (
+		mockCtrl         *gomock.Controller
+		mockOcmInterface *ocmMock.MockOCMInterface
+	)
+
+	ocmEnv, _ := cmv1.NewEnvironment().BackplaneURL("https://backplane.example.com").Build()
+
+	ocmEnvNil, _ := cmv1.NewEnvironment().BackplaneURL("").Build()
+
+	cmd := &cobra.Command{Use: "get"}
```

Comment:
The `cmd` variable could be initialized within the BeforeEach block to ensure that each test case starts with a fresh instance which could prevent potential side effects from previous tests.

Reply:
If i declare cmd inside before each block, it's not been exported as global variable. i.e, can't use it in the test case